src/components/Toast/Toast.tsx

```diff
@@ -39,7 +39,7 @@ export const isNotificationTypes = (type: string): type is NotificationTypes =>
   return ['success', 'error', 'warning', 'info'].includes(type);
 };
 
-const Toast: React.FCC<ToastProps> = ({
+const Toast: React.FC<React.PropsWithChildren<ToastProps>> = ({
```

Comment:
this could probably be FCC

Reply:
fixed here https://github.com/Orfium/orfium-ictinus/pull/718/commits/b8d5f77f8605a2c62ea17e696ae816d9a509dc14